src/complete.js

```diff
@@ -0,0 +1,20 @@
+/* eslint-disable import/prefer-default-export */
+import { saveChanges } from './drag.js';
+
+// Checks if input are checked or not, and Add Or remove Css Class => The line through style.
+const getContainer = (newContainer) => {
+  const d = [...newContainer.querySelectorAll('input')];
+  d.forEach((input) => {
+    input.addEventListener('change', (e) => {
+      if (e.target.checked) {
+        input.parentNode.classList.add('desc');
+        saveChanges();
+      } else {
+        input.parentNode.classList.remove('desc');
+        saveChanges();
+      }
```

Comment:
- You did a really nice work here to conditionally style it checked list item. But, your condition should not only be the check box, but also the status in the local storage. This is because if `completed` is true in the local storage and I refresh the page, the crossing line and the checkmark disappear, but the true `completed` status is true. So make your if condition to check for the local storage `completed` status too. See the screenshot to see the current behavior.
![Screenshot from 2021-07-22 04-53-39](https://user-images.githubusercontent.com/64894927/126570670-aa32cfc8-6524-4d45-ae55-5ada6f5788ba.png)